cli/main.rs

```diff
@@ -734,10 +734,53 @@ async fn test_command(
       .filter(|e| e.url.contains(cwd.as_os_str().to_str().unwrap()))
       .collect::<Vec<ScriptCoverage>>();
 
-    // TODO(caspervonb) print a summary report to console
     // TODO(caspervonb) add support for lcov output (see geninfo(1) for format spec).
+    println!("test coverage:");
 
-    write_json_to_stdout(&filtered_coverage)?;
+    for script in filtered_coverage {
+      let url = Url::parse(&script.url)?;
+      let path = url.to_file_path().unwrap();
+      let source = tokio::fs::read_to_string(&path).await?;
+
+      let mut total = 0;
+      let mut covered = 0;
+
+      let mut offset = 0;
+      for line in source.lines() {
+        let line_start_offset = offset;
+        let line_end_offset = line_start_offset + line.len();
```

Comment:
Appears that we need to look these up in the source map.

Should probably also use the source fetcher instead of tokio here.

Reply:
Correct, I suggest extracting this loop into a standalone function in `coverage.rs`. 

You can use `get_orig_position` from `source_maps.rs` to lookup correct position

Reply:
Actually that should be done using `GlobalState.ts_compiler.get_source_line()`